### DISCUSSION TITLE
SOLWEIG-mean radiant temperature is higher in forest land than in other lands

### DISCUSSION THREAD

**Discussion by land333land** [UMEP-dev/UMEP#552] [Q&A]
The Tmrt operates normally, but the results are unreasonable. At any time during the day, Tmrt is higher in forest land than in other lands. I'm not sure what is causing this problem.

**Comment by biglimp** [comment]
Its hard to know what is causing this. To begin with, I can see two things that you can check.

1. Is your result showing reasonable result without using vegetation (CDSM)?
2. Is your CDSM units in meter above ground level and is all non-vegetated pixels = 0?

**Reply by land333land** [reply]
Based on your suggestion, I have checked, but the results are still not reasonable. 

The CDSM data matches your description. Moreover, even though I did not use CDSM data in calculating Tmrt, the results are still unreasonable.
I checked the possible reasons for this result, and I think the SVF factor is a key reason, the value of SVF in the forest land is usually close to 0. The general knowledge is that there is a negative correlation between SVF and temperature, but the temperature in the forest land should not be higher than the impervious surface.

I'm not sure there could be any other reason for this result?

**Reply by biglimp** [reply]
SVF should be close to 0 in forests since SVF is calculated from underneath the canopies. Can you send me your data (metfile, dsm, dem, cdsm and landcover (if used) for me to have a look at.

**Reply by land333land** [reply]
I seem to have found the reason for this unreasonable result, and it may be due to my weather data. I may have to check the weather data again.
